app/config/AppConfig.scala

```diff
@@ -138,6 +147,28 @@ class AppConfigImpl @Inject() (config: ServicesConfig, configuration: Configurat
 
   def apiStatus(version: Version): String = config.getString(s"api.$version.status")
 
+  private val DATEFORMATTER = new DateTimeFormatterBuilder()
+    .append(DateTimeFormatter.ofPattern("yyyy-MM-dd"))
+    .parseDefaulting(ChronoField.HOUR_OF_DAY, 23)
+    .parseDefaulting(ChronoField.MINUTE_OF_HOUR, 59)
+    .parseDefaulting(ChronoField.SECOND_OF_MINUTE, 59)
+    .toFormatter()
+
+  def deprecatedOn(version: Version): Option[LocalDateTime] = configuration.getOptional[String](s"api.$version.deprecatedOn") match {
+    case Some(value) => Some(LocalDateTime.parse(value, DATEFORMATTER))
+    case None        => None
+  }
+
+  def sunsetDate(version: Version): Option[LocalDateTime] = configuration.getOptional[String](s"api.$version.sunsetDate") match {
+    case Some(value) => Some(LocalDateTime.parse(value, DATEFORMATTER))
+    case None        => None
+  }
+
+  override def sunsetEnabled(version: Version): Boolean = configuration.getOptional[Boolean](s"api.$version.sunsetEnabled") match {
+    case Some(value) => value
+    case None        => true
+  }
```

Comment:
```suggestion
  def deprecatedOn(version: Version): Option[LocalDateTime] = configuration.getOptional[String](s"api.$version.deprecatedOn").map(LocalDateTime.parse(value, DATEFORMATTER))

  def sunsetDate(version: Version): Option[LocalDateTime] = configuration.getOptional[String](s"api.$version.sunsetDate").map(LocalDateTime.parse(value, DATEFORMATTER))

  override def sunsetEnabled(version: Version): Boolean = configuration.getOptional[Boolean](s"api.$version.sunsetEnabled").getOrElse(true)
```

Reply:
The last one would be better to used the methods from `config` which will lead it to default to true anyway simplifying the code a tad